user-operator/src/main/java/io/strimzi/operator/user/model/acl/SimpleAclRule.java

```diff
@@ -180,6 +295,62 @@ public Acl toKafkaAcl(KafkaPrincipal principal)   {
         return new Acl(principal, kafkaType, getHost(), kafkaOperation);
     }
 
+    public static SimpleAclRule fromKafkaAccessControlEntry(SimpleAclRuleResource resource, AccessControlEntry accessControlEntry)   {
+        AclRuleType type;
+        AclOperation operation;
+
+        switch (accessControlEntry.permissionType()) {
+            case DENY:
+                type = AclRuleType.DENY;
+                break;
+            case ALLOW:
+                type = AclRuleType.ALLOW;
+                break;
+            default:
+                throw new IllegalArgumentException("Invalid AclRule type: " + accessControlEntry.permissionType());
+        }
+
+        switch (accessControlEntry.operation()) {
+            case READ:
+                operation = AclOperation.READ;
+                break;
+            case WRITE:
+                operation = AclOperation.WRITE;
+                break;
+            case CREATE:
+                operation = AclOperation.CREATE;
+                break;
+            case DELETE:
+                operation = AclOperation.DELETE;
+                break;
+            case ALTER:
+                operation = AclOperation.ALTER;
+                break;
+            case DESCRIBE:
+                operation = AclOperation.DESCRIBE;
+                break;
+            case CLUSTER_ACTION:
+                operation = AclOperation.CLUSTERACTION;
+                break;
+            case ALTER_CONFIGS:
+                operation = AclOperation.ALTERCONFIGS;
+                break;
+            case DESCRIBE_CONFIGS:
+                operation = AclOperation.DESCRIBECONFIGS;
+                break;
+            case IDEMPOTENT_WRITE:
+                operation = AclOperation.IDEMPOTENTWRITE;
+                break;
+            case ALL:
+                operation = AclOperation.ALL;
+                break;
+            default:
+                throw new IllegalArgumentException("Invalid AclRule operation: " + accessControlEntry.operation());
+        }
```

Comment:
Can comment here about having two separate methods for these.

Reply:
Ditto as above